### PR TITLE
Show a link instead of inline photo for photos served by other domains in zero-rating mode. #218

### PR DESCRIPTION
**Pull request by gimite**

### CODE REVIEW

File: app/resources/note.html.template [hunk 1]
```diff
@@ -161,7 +161,11 @@
     {% if note.photo_url %}
       <div>
         <a href="{{note.photo_url_no_scheme}}">
-          <img src="{{note.photo_url_no_scheme}}" class="photo">
+          {% if note.should_show_inline_photo %}
+            <img src="{{note.photo_url_no_scheme}}" class="photo">
+          {% else %}
+            {% trans "Photo (external site; normal connection fee may apply)" %}
```
Comment (skywhale) [inline app/resources/note.html.template]: View photo?

File: app/resources/view.html.template [hunk 2]
```diff
@@ -148,8 +148,12 @@
                 <div class="field">
                   <span>
                     <a href="{{person.photo_url_no_scheme}}">
-                      <img src="{{person.photo_url_no_scheme}}"
-                          class="photo" alt="[PHOTO]">
+                      {% if person.should_show_inline_photo %}
+                        <img src="{{person.photo_url_no_scheme}}"
+                            class="photo" alt="[PHOTO]">
```
Comment (skywhale) [inline app/resources/view.html.template]: pre-existing but we should remove brackets from alt.

**Comment by skywhale** [comment]
LGTM

**Comment by gimite** [comment]
Thanks. Merged.

**Pull request merged**